pkg/bundle/internal/target/target.go

```diff
@@ -442,6 +446,16 @@ func TrustBundleHash(data []byte, additionalFormats *trustapi.AdditionalFormats)
 		_, _ = hash.Write([]byte(*additionalFormats.PKCS12.Password))
 	}
 
+	// Add Target annotations and labels to the hash so it becomes aware of changes and triggers an update.
+	if target != nil {
```

Comment:
I don't think `target` should be nil in this context. But maybe make `GetAnnotations` and `GetLabels` nil-safe, to make this a bit more compact?